ui/packages/ui/src/app/components/formHelpers/FormSelectComponent.tsx

```diff
@@ -64,7 +64,7 @@ export const FormSelectComponent = (props: IFormSelectComponentProps) => {
   };
 
   const selectOptions = options.map((value: any) => {
-    return {'value': value.charAt(0).toUpperCase() + value.slice(1)}
+    return {'value': value.charAt(0) + value.slice(1)}
```

Comment:
I don't think you need to split the first character out anymore - just return the value?